xpm/fuzz_test.go

```diff
@@ -0,0 +1,32 @@
+package xpm
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+// FuzzParsePanic runs a fuzzer to detect possible panics in the XPM
+// parser.  To run, use:
+//
+//	go test -fuzz=FuzzParsePanic
+func FuzzParsePanic(f *testing.F) {
```

Comment:
I think it would be ok to run fuzz automatically everyday in a github action. I usually pick 4am as that's when the activity on fyne repository are at their lowest. If you do not mind adding a new action, I think that will be useful to run this fuzzing regularly.

Reply:
Fuzzing runs indefinitely if you let it, and I'm not sure whether Github actions are the best fit there.  (TBH, I'm not terribly familiar with Github actions either.)  I suspect that these considerations would inflate the scope of this change too much.  I'd rather leave it up to you to configure this, if that's OK with you. :)